packages/web/.env/.env.prod

```diff
@@ -19,6 +19,7 @@ VITE_WORMHOLE_ADDRESS=0x6E7a1F7339bbB62b23D44797b63e4258d283E095
 VITE_CLAIM_DISTRIBUTION_CONTRACT_ADDRESS=0x683c19E621A0F107a291fdAB38f80179809d61B5
 
 VITE_SOLANA_WEB3_CLUSTER=mainnet-beta
+VITE_SOLANA_RELAY_ENDPOINT=http://discoveryprovider.audius.co
```

Comment:
Just want to confirm this doesn't put us in any _worse_ of a position than hitting identity relay.
* Does this get load balanced at an arbitrary DN? i.e. if we have a DN down, will that prevent all relays from happening?
* If a user is connected to a DN in another country and then relaying to a DN in NorthAmerica, any concerns there? I would think no as this is the same arrangement we have with calling identity for relays as it's hosted in NA.